src/components/SignUp/CompleteSignupFlow.tsx

```diff
@@ -95,7 +96,7 @@ class CompleteSignupFlow extends Component<Props, State, {}> {
 
   handleChangeLastname = (e: { target: { value: string; }; }) => this.setState({ lastname: e.target.value });
 
-  handleChangeBirthdate = (date: Date, callback) => this.setState({ birthDate: date }, callback);
+  handleChangeBirthdate = (date: Date) => this.setState({ birthDate: date });
```

Comment:
why are you removing this callback? without it, the datepicker actually doesn't actually trigger if you change it